source/components/inputs/select/select.ng1.ts

```diff
@@ -96,10 +96,8 @@ export class SelectController extends InputController {
 		this.showOptions = !this.showOptions;
 	}
 
-	close: { (): void } = () => {
-		if (this.showOptions) {
```

Comment:
Why this change? If showOptions is already true, it shouldn't matter whether you clear it or not.